### PR TITLE
Update gfx942 NN/TN Equality/GridBased yamls for non llm sizes

### PR DESCRIPTION
**Pull request by AndySu12**

### CODE REVIEW

**Review by aazz44ss — DISMISSED** [review]
LGTM

**Review by hcman2 — DISMISSED** [review]
ok if CI passed

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/GridBased/aquavanjaram_Cijk_Ailk_Bljk_BBS_BH_Bias_AS_SAV_UserArgs.yaml [hunk 1]
```diff
@@ -526953,6 +533048,76 @@
     - [1780, 0.0]
   - - [67108864, 32, 1, 32]
     - [1782, 0.0]
+  - - [2048, 18848, 1, 512]
+    - [1783, 364595.0]
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/GridBased/aquavanjaram_Cijk_Ailk_Bljk_BBS_BH_Bias_AS_SAV_UserArgs.yaml:533052]: suggest cleaning the performance number

Comment (AndySu12) [reply library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/GridBased/aquavanjaram_Cijk_Ailk_Bljk_BBS_BH_Bias_AS_SAV_UserArgs.yaml:533052]: Done, thanks.

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by aazz44ss — APPROVED** [review]
LGTM

**Pull request merged**